wasm/module_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 The go-interpreter Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package wasm_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/go-interpreter/wagon/wasm"
+)
+
+func TestReadModule(t *testing.T) {
+	fnames, err := filepath.Glob(filepath.Join("testdata", "*.wasm"))
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, fname := range fnames {
+		testModule(t, fname)
+	}
+}
+
+func testModule(t *testing.T, fname string) {
+	f, err := os.Open(fname)
```

Comment:
👍 to subtests and bringing the actual test code back into `TestReadModule`. It's more readable now.